pkg/operator/ceph/object/rgw.go

```diff
@@ -145,6 +146,17 @@ func (c *clusterConfig) startRGWPods(realmName, zoneGroupName, zoneName string)
 			}
 		}
 
+		doMultisiteSync := "true"
+		if c.store.Spec.Gateway.DisableMultisiteSyncTraffic {
+			doMultisiteSync = "false"
+		}
+		monStore := cephconfig.GetMonStore(c.context, c.clusterInfo)
+		who := generateCephXUser(rgwConfig.ResourceName)
+		err = monStore.Set(who, "rgw_run_sync_thread", doMultisiteSync)
```

Comment:
Instead of setting this here, could you add it to the `setDefaultFlagsMonConfigStore()` method? Then that method could be renamed to `setFlagsMonConfigStore()` to more accurately represent that it applies not just default settings.

If you're feeling really ambitious, you could change the implementation in that method to call `monStore.SetAll()` instead of `monStore.Set()` to more efficiently set them at the same time. But that could be a separate PR if desired.

Reply:
I found some time to work on this again: what you propose is way cleaner! The tests still pass but I'll need to check the behavior again against a cluster to be sure.

I kept using `monStore.Set()`. I'll look at using `monStore.SetAll()` when I find some time.